### PR TITLE
[Doc] Add spark load sql statement doc and update manual

### PR DESCRIPTION
**Pull request by wyb**
1. add sql statement in dml
2. update spark load manual

#3433

### CODE REVIEW

File: docs/zh-CN/administrator-guide/load-data/spark-load-manual.md [hunk 1]
```diff
@@ -251,6 +252,14 @@ LOAD LABEL load_label
     [PARTITION (p1, p2)]
     [COLUMNS TERMINATED BY separator ]
     [(col1, ...)]
+    [COLUMNS FROM PATH AS (col2, ...)]
+    [SET (k1=f1(xx), k2=f2(xx))]
```
Comment (morningman) [inline docs/zh-CN/administrator-guide/load-data/spark-load-manual.md:256]: Do we support function now?

File: docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SPARK LOAD.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+            column_list：
+
+            用于指定导入文件中的列和 table 中的列的对应关系。
+            当需要跳过导入文件中的某一列时，将该列指定为 table 中不存在的列名即可。
+            语法：
+            (col_name1, col_name2, ...)
+            
+            SET:
+
+            如果指定此参数，可以将源文件某一列按照函数进行转化，然后将转化后的结果导入到table中。语法为 `column_name` = expression。举几个例子帮助理解。
```
Comment (morningman) [inline docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SPARK LOAD.md:116]: 我们现在应该还不支持函数吧？

**Review by morningman — COMMENTED** [review]
You need also update the sidebar file in `docs/.vuepress/sidebar/zh-CN.js` and `docs/.vuepress/sidebar/en.js`

File: docs/zh-CN/administrator-guide/load-data/spark-load-manual.md (hunk 1 above)
Comment (wyb) [reply docs/zh-CN/administrator-guide/load-data/spark-load-manual.md:256]: Already supported

File: docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SPARK LOAD.md (hunk 2 above)
Comment (wyb) [reply docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SPARK LOAD.md:116]: Already supported

File: docs/zh-CN/administrator-guide/load-data/spark-load-manual.md (hunk 1 above)
Comment (wyb) [reply docs/zh-CN/administrator-guide/load-data/spark-load-manual.md:256]: Already supported

File: docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SPARK LOAD.md (hunk 2 above)
Comment (wyb) [reply docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SPARK LOAD.md:116]: Already supported

**Comment by wyb** [comment]
> You need also update the sidebar file in `docs/.vuepress/sidebar/zh-CN.js` and `docs/.vuepress/sidebar/en.js`

Add in zh-CN.js, and i will add en doc after spark load is stable.

File: docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SPARK LOAD.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+            例1: 表中有3个列“c1, c2, c3", 源文件中前两列依次对应(c1,c2)，后两列之和对应c3；那么需要指定 columns (c1,c2,tmp_c3,tmp_c4) SET (c3=tmp_c3+tmp_c4); 
+            例2: 表中有3个列“year, month, day"三个列，源文件中只有一个时间列，为”2018-06-01 01:02:03“格式。
```
Comment (WuZixing) [inline docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SPARK LOAD.md:118]:
```suggestion
            例2: 表中有3个列“year, month, day"，源文件中只有一个时间列，为”2018-06-01 01:02:03“格式。
```

File: docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SPARK LOAD.md [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+
+        则提取文件路径的中的city和utc_date字段
+
+    5. 对待导入数据进行过滤，k1 值大于 10 的列才能被导入。
+
+        LOAD LABEL example_db.label10
+        (
+        DATA INFILE("hdfs://hdfs_host:hdfs_port/user/palo/data/input/file")
+        INTO TABLE `my_table`
+        where k1 > 10
```
Comment (WuZixing) [inline docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SPARK LOAD.md:244]:
```suggestion
        WHERE k1 > 10
```

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**